Tools/clinic/clinic.py

```diff
@@ -5308,23 +5308,13 @@ def state_function_docstring(self, line: str) -> None:
 
         self.docstring_append(self.function, line)
 
-    def format_docstring(self) -> str:
-        f = self.function
-        assert f is not None
-
-        new_or_init = f.kind.new_or_init
-        if new_or_init and not f.docstring:
-            # don't render a docstring at all, no signature, nothing.
-            return f.docstring
-
+    @staticmethod
+    def format_docstring_signature(
+            f: Function,
+            parameters: list[Parameter]
```

Comment:
the parameters here look over-indented:

```suggestion
        f: Function,
        parameters: list[Parameter]
```

Reply:
It's a personal preference.

Reply:
If you browse through clinic.py, you'll see my style and Alex's style :)

Reply:
I suspect my style is breaking with the norm in the typing community. I just like that the params are visually separated from the function/method name.

Reply:
> I suspect my style is breaking with the norm in the typing community.

I don't think the typing community has any particular preferences w.r.t. formatting compared to other parts of the Python ecosystem; I think it's more to do with the fact that most of the open-source ecosystem now uses `black` to autoformat their code 😄 and since I'm a fan of most of black's formatting choices, I tend to follow a pretty black-inspired formatting style even when I'm _not_ using black :)